config/pj-on-kind.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runs prow/pj-on-kind.sh with config arguments specific to the prow.k8s.io instance.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export CONFIG_PATH="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../prow/config.yaml)"
+export JOB_CONFIG_PATH="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/jobs)"
+bash <(curl -s https://raw.githubusercontent.com/kubernetes/test-infra/master/prow/pj-on-kind.sh) "$@"
```

Comment:
Why a `curl` instead of relative path?

Reply:
With `curl`, this script can be copied into any other repo with Prow config (e.g. `istio/test-infra`) and the only change needed to make everything work is to adjust the config envvars accordingly. Then, jobs for that Prow instance can be run with `./pj-on-kind.sh my-istio-job`. Fetching the `k/t-i/prow/pj-on-kind.sh` file each time ensures that the latest version is always used.

In this case a relative path is equivalent, but I want a copy-pasteable example.

Reply:
I changed to use a relative path, but left the commented out the `curl` version.

Reply:
Might want to just package it in a container image for re-use